DevCycle/Models/CustomData.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  CustomData.swift
+//  DevCycle
+//
+//  Created by Jonathan Norris on 2022-01-24.
+//  Copyright © 2022 Taplytics. All rights reserved.
+//
+
+import Foundation
+
+public enum CustomDataValue: Codable {
+    case string(_ value: String)
+    case number(_ value: NSNumber)
+    case boolean(_ value: Bool)
```

Comment:
are we not allowing CustomData to be nested with more objects?

Reply:
No we want to force it to be a flat object.